docs/source/models.rst

```diff
@@ -1304,129 +1304,105 @@ For more info, see:
 Deploy a ``python_function`` model on Microsoft Azure ML
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The :py:mod:`mlflow.azureml` module can package ``python_function`` models into Azure ML container images and deploy them as a webservice. Models can be deployed to Azure Kubernetes Service (AKS) and the Azure Container Instances (ACI)
-platform for real-time serving. The resulting Azure ML ContainerImage contains a web server that
-accepts the following data formats as input:
+The MLflow plugin `azureml-mlflow <https://pypi.org/project/azureml-mlflow/>`_ can deploy models to Azure ML, either to Azure Kubernetes Service (AKS) or Azure Container Instances (ACI) for real-time serving. 
+
+The resulting container image contains a web server that accepts the following data formats as input:
 
 * JSON-serialized pandas DataFrames in the ``split`` orientation. For example, ``data = pandas_df.to_json(orient='split')``. This format is specified using a ``Content-Type`` request header value of ``application/json``.
 
-* :py:func:`mlflow.azureml.deploy` registers an MLflow Model with an existing Azure ML workspace, builds an Azure ML container image and deploys the model to AKS and ACI. The `Azure ML SDK`_ is required in order to use this function. *The Azure ML SDK requires Python 3. It cannot be installed with earlier versions of Python.*
+Deployments can be generated using both the Python API or MLflow CLI. On both cases, a ``JSON`` configuration file has to be specified with the details of the deployment you want to achieve. If not indicated, then a default deployment is done using Azure Container Instances (ACI). You can check the full specification at `Deployment configuration schema <https://docs.microsoft.com/en-us/azure/machine-learning/reference-azure-machine-learning-cli#deployment-configuration-schema>`_. The deployment configuration file looks as follows:
 
-.. _Azure ML SDK: https://docs.microsoft.com/python/api/overview/azure/ml/intro?view=azure-ml-py
+.. rubric:: For an ACI deployment
 
-.. rubric:: Example workflow using the Python API
+.. code-block:: json
 
-.. code-block:: py
+    {
+      computeType: aci,
```

Comment:
CircleCI checks are failing for invalid json structure. Here's an example that you can use to refer to: https://github.com/mlflow/mlflow/blob/master/docs/source/rest-api.rst#L664

Reply:
Goit it. Thanks for the handy example! It seems keys need to be in double quotes!

Reply:
I typically debug docs changes locally by:
```
cd docs
make clean  # clears out the cache in '_build'
make html SPHINXOPTS="-W --keep-going -n"
```
There will be an absolute wall of text generated and the first run you do will take a solid 10 minutes to build the docker containers, but you'll get a pretty good idea of verbose reporting on issues without having to push changes to remote. 

That being said, sphinx is notorious for being hard to track down just WHY it's lexer can't parse the format of something... A lot of formatting for me is done by trying to find examples and doing a lot of trial and error. Spacing & indentation is key for formats to be accepted, so do keep that in mind.

Reply:
Thanks @BenWilson2 for the tips!